### ISSUE TITLE
Done, with errors!

### ISSUE THREAD

**Issue by mak-cs** [cowboy/synology-update-plex#40]
### Script output
```
Checking for a Plex Media Server update...

[ Enabling Plex Pass releases ]
Found Plex Token

[ Retrieving version data ]
Available version: 1.24.2.4973-2b1b51db9

Done, with errors!
```

### NAS Model
DS1517+

### DSM Version
DSM 7.0-41890

### Additional context
Auto Update Plex Media Server on Synology NAS Version v1.0.3

**Comment by pablorusso** [comment]
Seems to be fixed by https://github.com/cowboy/synology-update-plex/pull/36

**Comment by mak-cs** [comment]
fixed by #36, thanks for your support!
